src/Model/Customer/CustomerBillingAddress.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Billogram\Model\Customer;
+
+/**
+ * @author Ibrahim Hizeoui <ibrahimhizeoui@gmail.com>
+ */
+class CustomerBillingAddress
+{
+    /**
+     * @var string
+     */
+    private $careOf;
+
+    /**
+     * @var bool
+     */
+    private $useCareOfAsAttention;
+
+    /**
+     * @var string
+     */
+    private $streetAddress;
+
+    /**
+     * @var string
+     */
+    private $zipCode;
+
+    /**
+     * @var string
+     */
+    private $city;
+
+    /**
+     * @var string
+     */
+    private $country;
+
+    /**
+     *
+     * @param string $careOf
+     * @param bool   $useCareOfAsAttention
+     * @param string $streetAddress
+     * @param string $zipCode
+     * @param string $city
+     * @param string $country
+     */
+    public function __construct(string $careOf, bool $useCareOfAsAttention, string $streetAddress, string $zipCode, string $city, string $country)
+    {
+        $this->careOf = $careOf;
+        $this->useCareOfAsAttention = $useCareOfAsAttention;
+        $this->streetAddress = $streetAddress;
+        $this->zipCode = $zipCode;
+        $this->city = $city;
+        $this->country = $country;
+    }
+
+    /**
+     * @return string
+     */
+    public function getCareOf()
+    {
+        return $this->careOf;
+    }
+
+    /**
+     * @param string $careOf
+     */
+    public function setCareOf(string $careOf)
+    {
+        $this->careOf = $careOf;
```

Comment:
This will make this object mutable. We do not want mutable objects. We should ONLY do immutable objects.